DICOM/src/Microsoft.Health.Dicom.Anonymizer.Core/Processors/Model/EncryptFunctionTypes.cs

```diff
@@ -0,0 +1,13 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Dicom.Anonymizer.Core.Model
+{
+    public enum EncryptFunctionTypes
+    {
+        // RSA could be supported in the future
```

Comment:
>RSA [](http://example.com/codeflow?start=11&length=3)

you may consider this carefully. If you just want to encrypt some data, public key encryption is not recommended. it is mostly used as a key exchange mechanism instead of encrypting data

Reply:
this might be defined by standard.
content should be encrypted by AES and key is entryped with RSA.

BTW, is the EncrypFunction in this tool related with Encrypted Attributes Sequence described in [http://dicom.nema.org/dicom/2013/output/chtml/part15/chapter_E.html](http://dicom.nema.org/dicom/2013/output/chtml/part15/chapter_E.html)?

Reply:
Correct, If we support re-identify. we need support decryption. And the key for AES might needs RSA to encrypt for key exchange.

For now, we do not support re-identify but may support in the future.

Reply:
@qfyin 
As for Encrypted Attributes Sequence problem. We have confirmed with PM. The standard encryption is given in this link, but we do not follow it for now, and just encrypt in place.

Reply:
From my perspective, the standard encryption in this link is important for customers. And it is better to implement it in next release.